webapp/src/components/link_tooltip/link_tooltip.test.jsx

```diff
@@ -1,43 +1,63 @@
 import {getInfoAboutLink} from './link_tooltip';
 
 describe('getInfoAboutLink should work as expected', () => {
-    it('Should return the correct owner, repo, type, and number when given a valid GitLab href and hostname with all required information', () => {
+    it('Should return an object of correct owner, repo, type, and number when given a valid GitLab hostname and href with all required information', () => {
         const href = 'https://gitlab.com/mattermost/test/-/merge_requests/123';
         const hostname = 'gitlab.com';
-        const expected = ['mattermost', 'test', '-', 'merge_requests', '123'];
+        const expected = {
+            owner: 'mattermost',
+            repo: 'test',
+            type: 'merge_requests',
+            number: '123',
+        };
 
         const result = getInfoAboutLink(href, hostname);
-
         expect(result).toEqual(expected);
     });
 
-    it('Should return the correct owner, repo, type, and undefined number when given a valid GitLab href and hostname missing the number', () => {
+    it('Should return an empty object when given a valid GitLab hostname and href missing the number', () => {
         const href = 'https://gitlab.com/mattermost/test/-/merge_requests';
         const hostname = 'gitlab.com';
-        const expected = ['mattermost', 'test', '-', 'merge_requests'];
 
         const result = getInfoAboutLink(href, hostname);
-
-        expect(result).toEqual(expected);
+        expect(result).toEqual({});
     });
 
-    it('Should return the correct owner, repo, undefined type, and number when given a valid GitLab href and hostname missing the type', () => {
+    it('Should return an empty object when given a valid GitLab hostname and href missing the type', () => {
```

Comment:
Should return an empty object when given a valid GitLab hostname and href with missing the type parameter